testpackage/tests/Flowthrough_x_inflow_y_outflow_acc/Flowthrough_x_inflow_y_outflow_acc.cfg

```diff
@@ -38,9 +38,9 @@ y_min = -1.3e8
 y_max = 1.3e8
 z_min = -6.5e6
 z_max = 6.5e6
-
-t_max = 650
+t_max = 600
 dt = 2.0
```

Comment:
You probably don't want to do that but some of the explicit dt are obsolete since there is dynamic timestepping.

Reply:
"next time", resolving

Reply:
done.